accounts/admin.py

```diff
@@ -155,91 +108,145 @@ def clean_username(self):
 
 class FreesoundUserAdmin(DjangoObjectActions, UserAdmin):
     search_fields = ('=username', '=email')
-    actions = (disable_active_user, disable_active_user_preserve_sounds, )
+    actions = ()
     list_display = ('username', 'email')
     list_filter = ()
     ordering = ('id', )
     show_full_result_count = False
     form = AdminUserForm
```

Comment:
the `clean_username` method in this form doesn't use `username_taken_by_other_user`, therefore it won't check deleted users

Reply:
Addressed in 0816d32316556e2e6f1da7a94869c259b4c47bef